zetasql/fuzzing/protobuf/zetasql_expression_extractor.h

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef ZETASQL_FUZZING_ZETASQL_EXPRESSION_EXTRACTOR_H
+#define ZETASQL_FUZZING_ZETASQL_EXPRESSION_EXTRACTOR_H
+
+#include "absl/strings/str_cat.h"
+#include "gtest/gtest_prod.h"
+#include "zetasql/fuzzing/protobuf/zetasql_expression_grammar.pb.h"
+#include "zetasql/fuzzing/protobuf/parameter_grammar.pb.h"
+
+using zetasql_expression_grammar::Expression;
```

Comment:
Do not put using statements in header files. If you really want to, you can move these inside the class declaration, or just use the namespace-qualified names.

https://google.github.io/styleguide/cppguide.html#Aliases